Classes/ExplorerInterface/FLEXExplorerViewController.m

```diff
@@ -329,27 +329,27 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 
 - (void)updateOverlayAndDescriptionForObjectIfNeeded:(id)object
 {
-    NSUInteger indexOfView = [self.viewsAtTapPoint indexOfObject:object];
-    if (indexOfView != NSNotFound) {
-        UIView *view = self.viewsAtTapPoint[indexOfView];
-        NSValue *key = [NSValue valueWithNonretainedObject:view];
-        UIView *outline = self.outlineViewsForVisibleViews[key];
+    NSUInteger indexOfItem = [self.itemsAtTapPoint indexOfObject:object];
```

Comment:
This lookup fails because it's looking for a view (`object`) in the items array. One approach would be to pass the item in the `context` parameter of the KVO.